packages/rss-handler/src/index.ts

```diff
@@ -118,9 +118,16 @@ Sentry.GCPFunction.init({
 
 const signToken = promisify(jwt.sign)
 const parser = new Parser({
+  timeout: 60000, // 60 seconds
```

Comment:
😬

Reply:
@jacksonh the default timeout is 60 seconds. I just try to make it more clear here

Reply:
Yeah sounds good. I think long time out instead of failing fast and repeating is probably the right thing for RSS so this is fine. I just hate to see timeouts > 4s. :)